### ISSUE TITLE
Fix lint warnings

### ISSUE THREAD

**Issue by SiAdcock** [guardian/source#549]
**Describe the bug**

`typescript-eslint` is currently generating [151 warnings](https://github.com/guardian/source/runs/1182566249?check_suite_focus=true#step:4:293) on each build. The longer we fail to fix them, the more warnings we will accrus.

**To Reproduce**

```sh
$ git clone git@github.com:guardian/source.git
$ cd source
$ yarn
$ yarn validate
```

**Expected behavior**

There should be no warnings

**:warning: Important note**

It is not necessary to fix all these warnings in a single PR 😄

**Comment by Kingarturs** [comment]
Can i work on this? ✌

**Comment by SiAdcock** [comment]
@Kingarturs certainly, thanks so much! Let me know if you need any help 🎉

**Comment by SiAdcock** [comment]
Fixed in #552
